lang/en.json

```diff
@@ -1817,6 +1817,25 @@
 "DND5E.WeaponSimpleProficiency": "Simple",
 "DND5E.WeaponSimpleR": "Simple Ranged",
 "DND5E.Weight": "Weight",
+"DND5E.WeightUnit": {
+  "Label": "Weight Units",
+  "Kilograms": {
```

Comment:
"Kilograms" and "Megagrams" are not actual words. The plural forms are "Kilogram" and "Megagram".

Reply:
> "Kilograms" and "Megagrams" are not actual words. The plural forms are "Kilogram" and "Megagram".

I don't believe that this is correct. Kilograms and Megagrams are totally valid plural forms 👍